### PR TITLE
[Runtime] Allow for excluding pid from state

### PR DESCRIPTION
**Pull request by RobDolinMS**
Replaces #459

Signed-off-by: Rob Dolin (MSFT) <robdolin@microsoft.com>

### CODE REVIEW

File: runtime.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The value MAY be one of:
 
     Additional values MAY be defined by the runtime, however, they MUST be used to represent new runtime states not defined above.
 * **`pid`**: (int) is the ID of the container process, as seen by the host.
+The pid MAY be excluded or set to a value indicating empty or null if it is not seen by the host.
```
Comment (dqminh) [inline runtime.md]: i wonder if its useful to further clarified what it means by "pid is not seen by the host". I only understand this phrase when reading background of https://github.com/opencontainers/runtime-spec/pull/459/files.

Perhaps something like:

```
pid MAY be excluded when the container's kernel is isolated from the host / or the host does not manage the container pids
```

**Comment by hqhq** [comment]
@RobDolinMS Any specific reasons that you can't do rebase in https://github.com/opencontainers/runtime-spec/pull/459 ? I'll be better to do base there so we can keep good tracking.

File: runtime.md (hunk 1 above)
Comment (wking) [inline runtime.md]: nit: there's some [trailing whitespace][1] on the current line.

[1]: https://travis-ci.org/opencontainers/runtime-spec/jobs/193148351#L293

**Comment by crosbymichael** [comment]
Please rebase your original PRs and not create new ones.

Thanks! ;)

**Pull request closed without merging**